### PR TITLE
Adds check to keep observers from being recognized as servants of ratvar

### LINKED ISSUE #36789: clock cultists ghosts cant leave reebe
Issue reported from  Round ID: 85688 (/tg/Station Bagil [ENGLISH] [US-EAST] [100% FREE LAG])
i was a clock cultist
as ghost i kept being teled back to the ark if i try to teleport to station

### PR DESCRIPTION
**Pull request by xenon-difluoride**
:cl: Astatineguy12
fix: Ghosts of servants of Ratvar are no longer confined to Reebe like their corporal counterparts.
/:cl:

fixes #36789

### CODE REVIEW

**Comment by AlexTheSysop42** [comment]
oof messed up the title

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
Clockghosts shouldn't even be a thing.

The fix needs to be on `is_servant_of_ratvar` by having observers always return false

File: code/game/gamemodes/clock_cult/clock_cult.dm [hunk 1]
```diff
@@ -45,7 +45,7 @@ Credit where due:
 ///////////
 
 /proc/is_servant_of_ratvar(mob/M)
-	return istype(M) && M.mind && M.mind.has_antag_datum(/datum/antagonist/clockcult)
+	return istype(M) && M.mind && M.mind.has_antag_datum(/datum/antagonist/clockcult) && !isobserver(M)
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/gamemodes/clock_cult/clock_cult.dm]: you could probably just change 47 to be `(mob/living/M)` to get the desired effect without doing a check to see if it's a mob AND a check to see if it isn't a dead mob.

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]

File: code/game/gamemodes/clock_cult/clock_cult.dm (hunk 1 above)
Comment (AlexTheSysop42) [reply code/game/gamemodes/clock_cult/clock_cult.dm]: done

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**